pyatlan/model/packages/asset_export(basic).py

```diff
@@ -0,0 +1,290 @@
+from __future__ import annotations
+
+from typing import List, Optional
+
+from pyatlan.model.enums import CustomWorkflowPackage
+from pyatlan.model.packages.base.custom_package import AbstractCustomPackage
+from pyatlan.model.workflow import WorkflowMetadata
+
+
+class AssetExportPackage(AbstractCustomPackage):
```

Comment:
If we follow this:  https://github.com/atlanhq/atlan-python/pull/402/files#r1818828755, then the name would be `AssetExportBasic`.